lib/internal/test_runner/test.js

```diff
@@ -436,6 +532,8 @@ class Test extends AsyncResource {
     super('Test');
 
     let { fn, name, parent } = options;
+    // TODO(pmarchini): The plan has additional options that a user can set via t.plan.
```

Comment:
@cjihrig, wdyt?

Reply:
I would hold off on this. This is code that runs for every single test, so we should be careful with what we add support for here.